e2e/test/scenarios/question/saved.cy.spec.js

```diff
@@ -462,7 +462,7 @@ describe(
     it("should allow you to enable a webhook alert", () => {
       H.visitQuestion(ORDERS_COUNT_QUESTION_ID);
       cy.findByTestId("sharing-menu-button").click();
-      H.popover().findByText("Create alert").click();
+      H.popover().findByText("Create alerts").click();
```

Comment:
Why "create alerts"? I couldn't find any discussion of this on Slack. Just coming to this PR without context, I couldn't tell what this had to do with the Mantine migration.